src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,55 @@
+.clock {
+  position: absolute;
+
+  @include center-position;
+  @include clock-size;
+
+  border: 5px solid black;
+  border-radius: 50%;
+
+  background: url("../img/5-55539_transparent-broken-clock-clipart-transparent-background-clock-face.png") #fff no-repeat center;
+  background-size: cover;
+
+}
+
+.clock::before {
+  content: "";
+
```

Comment:
only recommendation to use nesting during elements styling

Reply:
Agree with @ArtemHulkoDev , use sass nesting